packages/salesforcedx-utils-vscode/test/unit/commands/mocks.ts

```diff
@@ -26,6 +26,7 @@ export class MockChannel {
   public show(column?: any, preserveFocus?: any) {}
   public hide(): void {}
   public dispose(): void {}
+  public replace(value: string): void {}
```

Comment:
test update required in order to bump the version of vscode we're using.  Basically bring out test classes that extend vscode types up to spec.